logdna/utils.py

```diff
@@ -1,13 +1,15 @@
 import json
 import socket
 
+
 def is_jsonable(obj):
     try:
         json.dumps(obj)
         return True
-    except:
+    except TypeError:
```

Comment:
I think it's safer to catch all here. According to [the docs](https://docs.python.org/3/library/json.html#json.dump), it could be a `ValueError`, `OverflowError` or `TypeError`.

Reply:
ah. good cactch